### ISSUE TITLE
eventcatalog generate seems to do nothing

### ISSUE THREAD

**Issue by otbe** [boyney123/eventcatalog#84]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/boyney123/eventcatalog/blob/master/CONTRIBUTING.md#reporting-new-issues).

### Description

I created a new project via `npx @eventcatalog/create-eventcatalog@latest my-dir`

If I add a custom generator plugin and try to run `npm run generate` it seems to do nothing at all.

Output is
```
$ npm run generate           

> eventcatalog@0.0.1 generate
> eventcatalog generate


> @eventcatalog/core@0.0.9 generate
> node scripts/generate.js
```

If I run the script manually via
`PROJECT_DIR=/my-dir node node_modules/@eventcatalog/core/scripts/generate.js` it also does nothing until I remove the if at https://github.com/boyney123/eventcatalog/blob/master/packages/eventcatalog/scripts/generate.js#L22

Afterwards the manual approach works 
```
$ PROJECT_DIR=/my-dir node node_modules/@eventcatalog/core/scripts/generate.js
Generating EventCatalog docs using: my-plugin-
```

but `npm run generate` still does nothing.

### Steps to reproduce

Not quite sure whats the most easy solution.

### Expected behavior

Will run my generator.

### Actual behavior

Does not run anything

### Your environment

- EventCatalog version used: 0.0.9
- Environment name and version: Node.js 17.3
- Operating system and version: macOs 12.1 on M1 mac

**Comment by otbe** [comment]
A small update from my side:
_Sometimes_ it is working. After a fresh `npm install` it sometimes works and sometimes not. Dont get why

**Comment by boyney123** [comment]
Thanks @otbe , will take a look at this, I'm just releasing the AsyncAPI plugin so will be in this area!

**Comment by boyney123** [comment]
@otbe thanks for raising this. Should be fixed in `@eventcatalog/core@0.0.10`

**Comment by otbe** [comment]
Works 👍
